### ISSUE TITLE
Fatal error while running `kubectl get ....`

### ISSUE THREAD

**Issue by hus787** [kubernetes/kubernetes#34232]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

BUG REPORT

**Kubernetes version** (use `kubectl version`):

```
Client Version: version.Info{Major:"1", Minor:"3", GitVersion:"v1.3.7", GitCommit:"a2cba278cba1f6881bb0a7704d9cac6fca6ed435", GitTreeState:"clean", BuildDate:"2016-09-12T23:15:30Z", GoVersion:"go1.6.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"4", GitVersion:"v1.4.0", GitCommit:"a16c0a7f71a6f93c7e0f222d961f4675cd97a46b", GitTreeState:"clean", BuildDate:"2016-09-26T18:10:32Z", GoVersion:"go1.6.3", Compiler:"gc", Platform:"linux/amd64"}
```

**Environment**:
- Running kubectl on macOS Sierra

**What happened**:
- Ran: `kubectl get hpa,pod,deployment,rc,service,node -owide`
- Got: 

```
fatal error: unexpected signal during runtime execution
[signal 0xb code=0x1 addr=0x15a8a398814c pc=0xecfb]

runtime stack:
runtime.throw(0x1a4b1e0, 0x2a)
    /usr/local/go/src/runtime/panic.go:547 +0x90
runtime.sigpanic()
    /usr/local/go/src/runtime/sigpanic_unix.go:12 +0x5a
runtime.unlock(0x234b320)
    /usr/local/go/src/runtime/lock_sema.go:107 +0x14b
runtime.(*mheap).alloc_m(0x234b320, 0x3, 0x37, 0x2725658)
    /usr/local/go/src/runtime/mheap.go:492 +0x314
runtime.(*mheap).alloc.func1()
    /usr/local/go/src/runtime/mheap.go:502 +0x41
runtime.systemstack(0x7fff5fbfe338)
    /usr/local/go/src/runtime/asm_amd64.s:307 +0xab
runtime.(*mheap).alloc(0x234b320, 0x3, 0x10000000037, 0xe99f)
    /usr/local/go/src/runtime/mheap.go:503 +0x63
runtime.(*mcentral).grow(0x234de00, 0x0)
    /usr/local/go/src/runtime/mcentral.go:209 +0x93
runtime.(*mcentral).cacheSpan(0x234de00, 0x2725658)
    /usr/local/go/src/runtime/mcentral.go:89 +0x47d
runtime.(*mcache).refill(0x2692960, 0xc800000037, 0x2725658)
    /usr/local/go/src/runtime/mcache.go:119 +0xcc
runtime.mallocgc.func2()
    /usr/local/go/src/runtime/malloc.go:642 +0x2b
runtime.systemstack(0x2342600)
    /usr/local/go/src/runtime/asm_amd64.s:291 +0x79
runtime.mstart()
    /usr/local/go/src/runtime/proc.go:1051

goroutine 1 [running, locked to thread]:
runtime.systemstack_switch()
    /usr/local/go/src/runtime/asm_amd64.s:245 fp=0xc820241d78 sp=0xc820241d70
runtime.mallocgc(0x3000, 0x164ed60, 0x0, 0xc8202a2630)
    /usr/local/go/src/runtime/malloc.go:643 +0x869 fp=0xc820241e50 sp=0xc820241d78
runtime.newarray(0x164ed60, 0x133, 0x10b00000085)
    /usr/local/go/src/runtime/malloc.go:798 +0xc9 fp=0xc820241e90 sp=0xc820241e50
runtime.growslice(0x11e2480, 0xc8202e0a00, 0x86, 0x86, 0x87, 0x0, 0x0, 0x0)
    /usr/local/go/src/runtime/slice.go:100 +0x2c1 fp=0xc820241f00 sp=0xc820241e90
regexp/syntax.(*compiler).quest(0xc820269900, 0x10b0000007f, 0x10b00000000, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:228 +0x21f fp=0xc820241fb8 sp=0xc820241f00
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202ef9d0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x17cc fp=0xc820242240 sp=0xc820241fb8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efa40, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc8202424c8 sp=0xc820242240
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efab0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820242750 sp=0xc8202424c8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efb20, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc8202429d8 sp=0xc820242750
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efb90, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820242c60 sp=0xc8202429d8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efc00, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820242ee8 sp=0xc820242c60
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efc70, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820243170 sp=0xc820242ee8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efce0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc8202433f8 sp=0xc820243170
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efd50, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820243680 sp=0xc8202433f8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efdc0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820243908 sp=0xc820243680
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efe30, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820243b90 sp=0xc820243908
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202efea0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820243e18 sp=0xc820243b90
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202eff10, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc8202440a0 sp=0xc820243e18
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202eff80, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820244328 sp=0xc8202440a0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2000, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc8202445b0 sp=0xc820244328
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2070, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820244838 sp=0xc8202445b0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f20e0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820244ac0 sp=0xc820244838
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2150, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820244d48 sp=0xc820244ac0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f21c0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820244fd0 sp=0xc820244d48
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2230, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820245258 sp=0xc820244fd0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f22a0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc8202454e0 sp=0xc820245258
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2310, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820245768 sp=0xc8202454e0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2380, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc8202459f0 sp=0xc820245768
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f23f0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820245c78 sp=0xc8202459f0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2460, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820245f00 sp=0xc820245c78
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f24d0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820246188 sp=0xc820245f00
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2540, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820246410 sp=0xc820246188
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f25b0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820246698 sp=0xc820246410
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2620, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820246920 sp=0xc820246698
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2690, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820246ba8 sp=0xc820246920
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2700, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820246e30 sp=0xc820246ba8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2770, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc8202470b8 sp=0xc820246e30
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f27e0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820247340 sp=0xc8202470b8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2850, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc8202475c8 sp=0xc820247340
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f28c0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820247850 sp=0xc8202475c8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2930, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820247ad8 sp=0xc820247850
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f29a0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820247d60 sp=0xc820247ad8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2a10, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820247fe8 sp=0xc820247d60
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2a80, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820248270 sp=0xc820247fe8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2af0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc8202484f8 sp=0xc820248270
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2b60, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820248780 sp=0xc8202484f8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2bd0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820248a08 sp=0xc820248780
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2c40, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820248c90 sp=0xc820248a08
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2cb0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820248f18 sp=0xc820248c90
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2d20, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc8202491a0 sp=0xc820248f18
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2d90, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820249428 sp=0xc8202491a0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2e00, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc8202496b0 sp=0xc820249428
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2e70, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820249938 sp=0xc8202496b0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2ee0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820249bc0 sp=0xc820249938
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2f50, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820249e48 sp=0xc820249bc0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f2fc0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024a0d0 sp=0xc820249e48
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3030, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024a358 sp=0xc82024a0d0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f30a0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024a5e0 sp=0xc82024a358
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3110, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024a868 sp=0xc82024a5e0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3180, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024aaf0 sp=0xc82024a868
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f31f0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024ad78 sp=0xc82024aaf0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3260, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024b000 sp=0xc82024ad78
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f32d0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024b288 sp=0xc82024b000
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3340, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024b510 sp=0xc82024b288
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f33b0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024b798 sp=0xc82024b510
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3420, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024ba20 sp=0xc82024b798
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3490, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024bca8 sp=0xc82024ba20
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3500, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024bf30 sp=0xc82024bca8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3570, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024c1b8 sp=0xc82024bf30
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f35e0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024c440 sp=0xc82024c1b8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3650, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024c6c8 sp=0xc82024c440
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f36c0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024c950 sp=0xc82024c6c8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3730, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024cbd8 sp=0xc82024c950
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f37a0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024ce60 sp=0xc82024cbd8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3810, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024d0e8 sp=0xc82024ce60
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3880, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024d370 sp=0xc82024d0e8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f38f0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024d5f8 sp=0xc82024d370
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3960, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024d880 sp=0xc82024d5f8
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f39d0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024db08 sp=0xc82024d880
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3a40, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024dd90 sp=0xc82024db08
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3ab0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024e018 sp=0xc82024dd90
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3b20, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024e2a0 sp=0xc82024e018
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3b90, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024e528 sp=0xc82024e2a0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3c00, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024e7b0 sp=0xc82024e528
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3c70, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024ea38 sp=0xc82024e7b0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3ce0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024ecc0 sp=0xc82024ea38
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3d50, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024ef48 sp=0xc82024ecc0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3dc0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024f1d0 sp=0xc82024ef48
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3e30, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024f458 sp=0xc82024f1d0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3ea0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024f6e0 sp=0xc82024f458
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3f10, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024f968 sp=0xc82024f6e0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f3f80, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc82024fbf0 sp=0xc82024f968
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f4000, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc82024fe78 sp=0xc82024fbf0
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f4070, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820250100 sp=0xc82024fe78
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f40e0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820250388 sp=0xc820250100
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f4150, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820250610 sp=0xc820250388
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f41c0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820250898 sp=0xc820250610
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f4230, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820250b20 sp=0xc820250898
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f42a0, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:156 +0x1ab3 fp=0xc820250da8 sp=0xc820250b20
regexp/syntax.(*compiler).compile(0xc820269900, 0xc8202f4310, 0x0)
    /usr/local/go/src/regexp/syntax/compile.go:146 +0x1783 fp=0xc820251030 sp=0xc820250da8
...additional frames elided...

goroutine 5 [chan receive]:
k8s.io/kubernetes/vendor/github.com/golang/glog.(*loggingT).flushDaemon(0x23421e0)
    /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/golang/glog/glog.go:879 +0x67
created by k8s.io/kubernetes/vendor/github.com/golang/glog.init.1
    /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/golang/glog/glog.go:410 +0x297
```
- Ran again: Didn't happen.

**How to reproduce it** (as minimally and precisely as possible):

Can't reproduce. Reporting here just so that it gets registered. Probably Golang thingy (https://github.com/golang/go/issues/16331).

**Comment by AdoHe** [comment]
should be the same as #33823

**Comment by smarterclayton** [comment]
Fixed with Go 1.7
